modules/home-manager/globals.nix

```diff
@@ -11,5 +11,9 @@
       default = "teal";
       description = "Global Catppuccin accent";
     };
+
+    enable = (lib.mkEnableOption "Catppuccin globally") // {
+      default = config.catppuccin.enable;
+    };
```

Comment:
is this not infinite recursion
```suggestion
    enable = lib.mkEnableOption "Catppuccin globally";
```

Reply:
That works fine here.

Reply:
Removing it doesn't have any impact, I'll remove it.